torch/csrc/jit/python/init.cpp

```diff
@@ -1073,8 +1070,7 @@ void initJITBindings(PyObject* module) {
                           self_func.ptr(),
                           module_name.c_str()));
                 }
-                return invokeOperatorFromPython(
-                    operations, std::move(args), std::move(kwargs));
+                return invokeOperatorFromPython(operations, args, kwargs);
```

Comment:
what's changed?

Reply:
So changing this file resulted in clang-tidy tool to be run on the init.cpp file.
I saw this error from the tool:
{
    path: 'torch/csrc/jit/python/init.cpp',
    start_line: 1074,
    end_line: 1074,
    start_column: 50,
    end_column: 50,
    annotation_level: 'failure',
    message: '[performance-move-const-arg] warning: passing result of std::move() as a const reference argument; no move will actually happen'
  }
  
  I believe args ang kwargs are const references, hence move() is not needed.

https://github.com/pytorch/pytorch/runs/1964096952